pygmt/tests/test_plot.py

```diff
@@ -452,6 +452,25 @@ def test_plot_datetime():
     return fig
 
 
+@pytest.mark.mpl_image_compare
+def test_plot_timedelta64():
+    """
+    Test plotting numpy.timedelta64 input data.
+    """
+    fig = Figure()
+    fig.basemap(
+        projection="X8c/5c",
+        region=[0, 8, 0, 10],
```

Comment:
Ideally, we could pass in `np.timedelta64` into the region argument as mentioned at https://github.com/GenericMappingTools/pygmt/issues/2848#issuecomment-1858698289.

```suggestion
        region=[np.timedelta64(0, "D"), np.timedelta64(8, "D"), 0, 10],
```

Reply:
Ah ok, looks like I modified `kwargs_to_strings` 3 years ago at #562 to handle `datetime` objects in `region`, so could probably do something similar here for `timedelta64`.